# macports:lang/sbcl 'gcc' should "just work"
# so remove reference to compiler in source configuration
--- src/runtime/Config.x86-64-darwin.old	2012-02-29 21:26:12.000000000 +0100
+++ src/runtime/Config.x86-64-darwin	2012-02-29 21:26:41.000000000 +0100
@@ -31,7 +31,6 @@
   OS_LIBS += -lz
 endif
 
-CC = gcc
 
 ASSEM_SRC = x86-64-assem.S ldso-stubs.S
 ARCH_SRC = x86-64-arch.c
